aptos-node/Cargo.toml

```diff
@@ -79,6 +79,7 @@ url = { workspace = true }
 
 [target.'cfg(unix)'.dependencies]
 jemallocator = { workspace = true }
+rstack-self = { workspace = true }
```

Comment:
How about using the `backtrace` crate - the API looks much simpler to me - https://crates.io/crates/backtrace

Reply:
That's for a different purpose, we need dump for every threads here.

Reply:
It support dumping for every threads no?

Reply:
no